VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/switch-on-camera-microphone/switch-on-camera-microphone.component.ts

```diff
@@ -72,15 +73,25 @@ export class SwitchOnCameraMicrophoneComponent extends ParticipantStatusBaseDire
     }
 
     async requestMedia() {
-        this.mediaAccepted = await this.userMediaStreamService.requestAccess();
-        this.userPrompted = true;
-        if (!this.mediaAccepted) {
-            this.logger.warn(`[SwitchOnCameraMicrophone] - ${this.participantName} denied access to camera.`, {
-                conference: this.conferenceId,
-                participant: this.participantName
+        this.userMediaStreamService.currentStream$
```

Comment:
does this need to be un subscribed as well or using the take operator resolves disposing this?

Reply:
"first" will handle that. https://stackoverflow.com/questions/42345969/take1-vs-first